Development.podspec

```diff
@@ -0,0 +1,91 @@
+Pod::Spec.new do |s|
+  s.name         = 'Development'
+  s.version      = '0.1.0'
+  s.summary      = 'Infrastructure for Workflow-powered UI'
+  s.homepage     = 'https://www.github.com/square/workflow'
+  s.license      = 'Apache License, Version 2.0'
+  s.author       = 'Square'
+  s.source       = { :git => 'https://github.com/square/workflow.git', :tag => "v#{s.version}" }
+
+  s.ios.deployment_target = '10.0'
+  s.swift_version = '5.0'
+  s.dependency 'Workflow'
+  s.dependency 'WorkflowUI'
+  s.source_files = 'swift/Samples/Dummy.swift'
+
+  s.subspec 'Dummy' do |ss|
+  end
+
+  s.default_subspecs = 'Dummy'
+
+  dir = Pathname.new(__FILE__).dirname
+
+  s.scheme = { 
```

Comment:
Do we need the environment variables in both places? What happens when we add a second Sample App with their own Reference Images? We could handle that as a fast follow, but eventually I'll be merging my vertical stack container screen which also will have snapshot tests.

https://github.com/square/workflow/pull/964/files#diff-fd7235628a2b67d5e50864050de83789R75

Reply:
We need them in both places if we want to be able to run these tests in both the `Development` and `Development-Unit-SplitScreenTests` schemes. We could remove them from this scheme but then switching to the `Development-Unit-SplitScreenTests` will be required.

That said, I can pull them out into a variable so they aren't defined twice